css/blog-style.css

```
@@ -90,11 +91,124 @@ body {
 }
 
 .main-container {
-    margin-top: 4rem;
+    margin-top: 5rem;
 }
 
-.text-center {
+/*BLOG IMAGE AND TITLE BEGINS  */
+
+.Blog-Title {

```

Keep class names lowercase
```suggestion
.blog-title {
```